pkg/manager/member/tikv_member_manager.go

```diff
@@ -318,9 +319,12 @@ func getNewTiKVSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap)
 	}
 
 	annoMount, annoVolume := annotationsMountVolume()
+	tikvDataVol := corev1.VolumeMount{
+		Name:      v1alpha1.TiKVMemberType.String(),
+		MountPath: tikvDataVolumeMountPath}
 	volMounts := []corev1.VolumeMount{
 		annoMount,
-		{Name: v1alpha1.TiKVMemberType.String(), MountPath: tikvDataVolumeMountPath},
+		tikvDataVol,
```

Comment:
no need to extract this one out, maybe format these 3 struct literal for better readability?

Reply:
The new containers need to use tikvDataVol after it.